specs/Standards/Sould Bound Token/README.md

```diff
@@ -0,0 +1,235 @@
+    fn sbt_token_for_owner(&self, account: AccountId) -> Option<Token> {}
+
+    /**********
+    * TRANSACTIONS
+    **********/
+
+    /// creates a new, unique token and assigns it to the `receiver`.
+    #[payable]
+    fn sbt_mint(&mut self, metadata: TokenMetadata, receiver: AccountId);
+
+    /// sbt_recover reassigns all tokens from the old owner to a new owner,
+    /// and registers `old_owner` to a burned addresses registry.
+    /// Must be called by operator.
+    /// Must provide 5 miliNEAR to cover registry storage cost. Operator should
+    ///   put that cost to the requester (old_owner), eg by asking operation fee.
+    #[payable]
+    fn sbt_recover(&mut self, from: AccountId, to: AccountId);
+
+    /// sbt_renew will update the expire time of provided tokens.
+    /// `expires_at` is a unix timestamp (in seconds).
+    #[payable]
+    pub fn sbt_renew(&mut self, tokens: Vec<TokenId>, expires_at: u64, memo: Option<String>);
+}
+```
+
+### Logs
+
+```typescript
+interface SbtEventLogData {
+  standard: "nepXXX";
+  version: "1.0.0";
+  event: "sbt_mint" | "sbt_recover" | "sbt_renew";
+  data: SbtMintLog[] | SbtRecoverLog[] | SbtRenew[];
+}
+
+// An event emitted when a new SBT is minted.
+// Arguments
+// * `owner`: "account.near"
+// * `tokens`: ["1", "abc"]
+// * `memo`: optional message
+interface SbtMintLog {
+  owner: string;
+  tokens: string[];
+  memo?: string;
+}
+
+// An event emitted when existing SBTs are revoked and burned.
+// Arguments
+// * `owner`: "account.near"
+// * `tokens`: ["1", "abc"]
+// * `memo`: optional message
+interface SbtRevokeLog {
+  owner: string;
+  tokens: string[];
+  memo?: string;
+}
+
+// An event emitted when a recovery process succeeded to reassign SBT.
+// Arguments
+// * `old_owner`: "old_account.near"
+// * `new_owner`: "new_account.near"
+// * `token_ids`: ["1", "abc"]
+// * `memo`: optional message
+interface SbtRecoverLog {
+  old_owner: string;
+  new_owner: string;
+  tokens: string[];
+  memo?: string;
+}
+
+// An event emitted when a existing tokens are renewed.
+// Arguments
+// * `tokens`: ["1", "abc"]
+// * `memo`: optional message
+interface SbtRenewLog {
+  tokens: uint64[];
+  memo?: string;
+}
+```
+
+Whenever a recovery is made in a way that an existing SBT is burned, the `SbtRevokeLog` event must be emitted.
+
+```typescript
+class SbtBurnedAccounts {
+  burn(account_id) {
+    this.burned_accounts[this.account_id][this.predecessor_id] = true;
+  }
+
+  // query
+  is_burned(account_id, sbt_token_contract): bool {
+    return this.burned_accounts[account_id][sbt_token_id];
+  }
+}
+```
+
+### Recommended functions
+
+Although the functions below are not part of the standard (depending on a use case, they may need different parameters), we recommend them as a part of implementation and we also provide them in the reference implementation.
+These functions should emit appropriate events.
+
+```typescript=
+
+interface SBT {
```

Comment:
- Again, shouldn't we prefix the methods with `sbt_`?
- Shouldn't we have `sbt_is_revoked` as a part of this interface?
- Can't this replace l. 172-181? Only diff I see is revoking by account vs. revoking by token_id.

Reply:
* I've reworked this interface, and moved transactions here.
* I don't think we should have `sbt_is_revoked` -- we would need to store per contract a registry of revoked tokens. There could be many implementations (eg burning a token, changing the expire time...).